### PR TITLE
Use solidity v0.6.10

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: cache/last-solc-config.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "solc": {
-    "version": "0.5.17",
+    "version": "0.6.8",
```
Comment (orenyodfat) [inline cache/last-solc-config.json:3]: use 0.6.10

**Comment by orenyodfat** [comment]
please handle` Warning: SPDX license identifier not provided in source file` 
https://forum.openzeppelin.com/t/solidity-0-6-8-introduces-spdx-license-identifiers/2859

File: contracts/libs/RealMath.sol [hunk 2]
```diff
@@ -13,17 +14,17 @@ pragma solidity ^0.5.17;
 
 library RealMath {
 
-    /**
+    /*
```
Comment (orenyodfat) [inline contracts/libs/RealMath.sol:17]: I think it should be` //` and not `/*`
https://solidity.readthedocs.io/en/v0.6.10/layout-of-source-files.html#comments

Comment (ben-kaufman) [reply contracts/libs/RealMath.sol:17]: But the doc you send said multi-line comments should be done with `/**/`

Comment (orenyodfat) [reply contracts/libs/RealMath.sol:17]: though this is not multi comment ..

Comment (ben-kaufman) [reply contracts/libs/RealMath.sol]: ok, I don't think this really matters but anyway I changed that

**Pull request merged**